source/usage-examples.txt

```diff
@@ -61,6 +61,38 @@ with your instance of MongoDB:
    const uri =
       "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&writeConcern=majority";
 
+All examples use ES module imports. You 
+`must enable ES module imports <https://nodejs.org/api/esm.html#esm_enabling>`__
+by adding the following to your package.json file:
```

Comment:
```suggestion
by adding the following code to your package.json file:
```

Reply:
Interesting comment! I'm not sure if the contents of a json file can reasonably be called "code" (I don't think it is executable if that is a good definition of code?), but you are right that it is ambiguous what is being referred to. Adding "you must add the following key-value pair to your..."